sympy/integrals/tests/test_integrals.py

```diff
@@ -1117,5 +1117,15 @@ def test_issue_4950():
         -2.4*exp(8*x) - 12.0*exp(5*x)
 
 
+def test_issue_10434():
+    w = symbols('w' , positive = True)
```

Comment:
It seems that the integral is evaluated even in current master if ``positive=True`` is assumed.